### ISSUE TITLE
Primitive Graphic Objects

### ISSUE THREAD

**Issue by csingewald** [syncpoint/ODIN#128]
support primitive graphic objects e.g. lines, polylines, polygons, circle, rectangle, text

**Comment by dehmer** [comment]
To clarify: Here we are talking about arbitrary features which are not related to tactical graphics.

**Comment by ThomasHalwax** [comment]
Yes, that is the intention. Those graphics should also support basic styling like
* color
* line style
